### PR TITLE
Assignment 2 Completed 

### PR DESCRIPTION
**Pull request by TechGlue**

### CODE REVIEW

**Comment by Ashrom0** [comment]
I will review this

**Comment by TechGlue** [comment]
Not too sure the reasons why the build would not work from the usal commands but in order to get it running on your local machine. Need to do a "npm install" then a "npm run build:dev" and then you can do a "dotnet run" to get it built and running

**Review by Ashrom0 — COMMENTED** [review]
Create CRUD pages for the following components
Users
A user only needs to have a First and Last name: ✔
Groups
A group will only need to hold the name of the group: ✔
Gifts
A gift should consist of the following information
Title: ✔
Description: ✔
Url: ✔
Priority (just a number for holding "order of importance"): ✔
The gift should hold a reference to the User it belongs to: ✔
A Home page that can be used as just a standard landing page for the application: ✔
Each page should have the ability to navigate between components: ✔

Nice work on the Assignment 2 hw

File: SecretSanta/src/SecretSanta.Web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
Comment (breaman) [inline SecretSanta/src/SecretSanta.Web/Views/Home/Index.cshtml:1]: This page should be using the _Layout file (which it is and is why you have navigation) but now in the body element of that page has html elements that it shouldn't.

**Review by breaman — COMMENTED** [review]
Overall, nice work on the assignment, really only have one issue with it in the Home/Index.cshtml file.

**Pull request closed without merging**